java-lib/src/main/java/com/wavefront/data/Validation.java

```diff
@@ -330,7 +335,14 @@ public static void validateHistogram(ReportHistogram histogram,
     }
   }
 
-  public static void validateSpan(Span span, @Nullable ValidationConfiguration config) {
```

Comment:
Keep a version of the method with the old method signature too, for backwards compatibility.